config/initializers/inflections.rb

```diff
@@ -12,6 +13,6 @@
 # end
 
 # These inflection rules are supported but not enabled by default:
-# ActiveSupport::Inflector.inflections(:en) do |inflect|
-#   inflect.acronym 'RESTful'
-# end
+ActiveSupport::Inflector.inflections(:en) do |inflect|
+  inflect.irregular "square foot", "square feet"
```

Comment:
I think the string that needs correcting is "foot", which appears to pluralize as "foots". It being preceded by "square" is irrelevant. So:
```ruby
inflect.irregular "foot", "feet"
```

Reply:
Thanks @reggieb 👍  we'll take another look at this.  However, I'm sure when we tested this as 'foot' and 'feet' we kept getting 'foots'.  But, with 'square foot' and 'square feet' it worked as expected.

Reply:
Looks like you are correct @reggieb 👍  I must have goofed somehow!  We will fix this.